sos/plugins/block.py

```diff
@@ -32,6 +41,9 @@ def setup(self):
             "ls -lanR /sys/block"
         ])
 
+        if self.get_option("all"):
```

Comment:
It always seems a good idea, until suddenly half the functionality is hidden behind magic options that most users and support engineers are unaware of: then you only get half the data by default, and have to make additional round-trips to request all the extra options be turned on.

There are very few reasons to make something non-default in `sos`:

* It really is very expensive or of limited use
* It requires authentication, special files, or other set up before it is useful and those are not guaranteed to be present

If it satisfies both of those, and it's still so slow you want to consider making it optional it's usually a sign that the feature needs to be re-thought, or some of the work moved out of sos (e.g. having a tool to do the processing for us, rather than text scraping and processing in Python - but those cases are rare).